notebook_sandbox/horne_extract/requirements.txt

```diff
@@ -2,10 +2,7 @@ astropy=5.0
 matplotlib==3.5.0
 numpy==1.21.4
 
-# remember the following:
+# KOSMOS-specific steps for notebook:
 # git clone git@github.com:jradavenport/kosmos.git OR git clone https://github.com/jradavenport/kosmos
-# fix ".data" error in trace function
-# use location as argument to sys.path.insert()
-
-# will also need to clone specreduce
-# git clone git@github.com:astropy/specreduce.git OR git clone https://github.com/astropy/specreduce
+# find and delete all occurrences of ".data" in kosmos/apextract.py
```

Comment:
note that if `NDData` or `CCDData` is used as an input, like me and @eteq have requested, this wouldn't be required. the docstring in `kosmos` says `2d numpy array, or CCDData object`, but the use of the `.data` attribute means it actually needs something `NDData`-like.

Reply:
I tried this, but it doesn't work because the `KOSMOS` trace also tries to call `img.shape` on the object, which isn't an `NDData` attribute. The best solution would be to update this notebook to use `KosmosTrace` once #86 is finished.